scripts/overview.js

```diff
@@ -100,18 +108,14 @@ function get_thumbnail () {
   fetch(new_url)
     .then(function (response) {
       $('#loader_thumbnail').hide()
-      if (response.size && response.size !== 233) {
-        $('#show_thumbnail').append($('<img>').attr('src', new_url))
-      } else {
-        $('#show_thumbnail').text('Thumbnail not found')
-      }
+      $('#show_thumbnail').append($('<img>').attr('src', new_url))
     })
     .catch(function (exception) {
       $('#loader_thumbnail').hide()
       if (exception === 'timeout') {
         $('#show_thumbnail').text('Please refresh the page...Time out!!')
       } else {
-        if (response.status === 504) { // TODO: FIXME: response not defined
+        if (exception.status === 504) { // TODO: FIXME: response not defined
```

Comment:
The code looks good.
Can we remove this 'TODO' comment?

Reply:
Yes actually, Still I am not sure of what is the exact response when the API call is failing or getting success !! For now we can delete it !

Reply:
Okay, thanks :-)